validator/api/proto/validator_api.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2018-2022 MobileCoin, Inc.
+
+// MUST BE KEPT IN SYNC WITH RUST CODE!
+
+// ledger validator node data types and service descriptors.
+
+syntax = "proto3";
+import "blockchain.proto";
+import "consensus_common.proto";
+import "external.proto";
+import "google/protobuf/empty.proto";
+import "report.proto";
+
+package validator_api;
+
+service ValidatorAPI {
```

Comment:
I believe these are the new APIs we will need to implement, in addition to BlockchainAPI.